src/cascadia/TerminalApp/TerminalPage.xaml

```diff
@@ -107,6 +107,20 @@
                     <Run x:Uid="AboutDialog_VersionLabel" />
                     <Run Text="{x:Bind ApplicationVersion}" />
                 </TextBlock>
+                <StackPanel Orientation="Vertical">
+                    <StackPanel Orientation="Horizontal" Visibility="{x:Bind CheckingForUpdates, Mode=OneWay}" Padding="0,4,0,4" VerticalAlignment="Center">
+                        <mux:ProgressRing Width="16" Height="16" IsIndeterminate="True" IsActive="True" />
+                        <TextBlock Text="Checking for updates..." Padding="4,0,0,0"/>
+                    </StackPanel>
+                    <StackPanel Orientation="Vertical" Visibility="{x:Bind UpdatesAvailable, Mode=OneWay}" Padding="0,4,0,4" VerticalAlignment="Center">
+                        <TextBlock IsTextSelectionEnabled="True">
+                            <Run Text="An update is available." /> <LineBreak />
+                            <Run x:Uid="AboutDialog_VersionLabel" />
+                            <Run Text="{x:Bind PendingUpdateVersion, Mode=OneWay}" />
+                        </TextBlock>
+                        <Button Content="Install now" Margin="0"></Button>

```

Comment:
.... Do we know how to force the store to install an update for us? [`StoreContext.RequestDownloadAndInstallStorePackageUpdatesAsync`](https://docs.microsoft.com/en-us/uwp/api/windows.services.store.storecontext.requestdownloadandinstallstorepackageupdatesasync?view=winrt-22621#windows-services-store-storecontext-requestdownloadandinstallstorepackageupdatesasync(windows-foundation-collections-iiterable((windows-services-store-storepackageupdate)))) presumably? Can we even locally test if that works?

Reply:
If you mean without user confirmation (silently) this could help according to docs: https://docs.microsoft.com/en-us/windows/uwp/packaging/self-install-package-updates#download-and-install-package-updates-silently

But why didn't you only opening the updates page in store? And what happened if stire isn't available/removed?